packages/account/src/Sections/Security/ConnectedApps/data-list-template-entry.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+
+type TDataListTemplateEntry = {
+    title: JSX.Element | string;
+    content: JSX.Element | string;
+};
+
+const DataListTemplateEntry = ({ title, content }: TDataListTemplateEntry) => {
+    const text_size = 'xxs';
+
+    return (
+        <React.Fragment>
+            <Text size={text_size} weight='bold'>
```

Comment:
**NIT**: If we need not provide the size based on device, can we directly assign the text_size rather than using variable?

Reply:
of course. 🔥